internal/mode/static/nginx/config/upstreams.go

```diff
@@ -35,6 +35,24 @@ func (g GeneratorImpl) executeUpstreams(conf dataplane.Configuration) []executeR
 	return []executeResult{result}
 }
 
+func (g GeneratorImpl) executeStreamUpstreams(_ dataplane.Configuration) []executeResult {
+	upstreams := []http.Upstream{
```

Comment:
Since stream upstreams are their own [module](https://nginx.org/en/docs/stream/ngx_stream_upstream_module.html) in nginx, I think it makes sense to create a new type for them. I know right now it will look identical to the http upstreams, but eventually they will be different.